src/Symfony/Component/Form/AbstractType.php

```diff
@@ -64,8 +64,16 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
+    public function getName()
+    {
+        return;
```

Comment:
I think you can remove getName form FormTypeInterface and just check if method_exists whenever we need call the method for BC reasons.